library/core/src/ptr/non_null.rs

```diff
@@ -587,7 +585,6 @@ impl<T: ?Sized> NonNull<T> {
     #[stable(feature = "non_null_convenience", since = "1.80.0")]
     #[rustc_const_stable(feature = "non_null_convenience", since = "1.80.0")]
     #[kani::requires(
-        count <= (isize::MAX as usize) &&
```

Comment:
Why was this removed? I think this is correct since `count` is a `usize`.

Reply:
Sorry, I delete this line by mistake. Already add it back in latest commit